js/index.js

```diff
@@ -0,0 +1,43 @@
+const city = 'London'; //デフォルト
+// API呼び出す
+async function callApi(city = 'London') {
```

Comment:
```suggestion
const callApi = async (city = 'London') => {
```

functionキーワードを廃止してアロー関数にしましょう。

async/await を使っているのはよいですね！